### DISCUSSION TITLE
FCM Notifications Cannot diplay image

### DISCUSSION THREAD

**Discussion by gian-almada** [firebase/flutterfire#4992] [Q&A]
i have working push notification using FCM, everything is okay until i used notification with image, its always shown something like this, 

![image](https://user-images.githubusercontent.com/42395245/107327935-874b2700-6ae0-11eb-9a05-4becdbd867a7.png)

what should i do ? thanks.

**Comment by miqueasgutierrez** [comment]
Hello, Test with the INDIGITALL Service, it is also Free for 30000 Devices, it allows to send Push Notifications with also geo-located images, programmed among many more functions, here is the link of the documentation to install it easily: https: //docs.indigitall .you eat/
